packages.json

```diff
@@ -23156,5 +23156,22 @@
     "description": "A Nim client for Todoist's REST API",
     "license": "Apache-2.0",
     "web": "https://ruivieira.github.io/nim-todoist/index.html"
+  },
+  {
+    "name": "pl0t",
+    "url": "https://github.com/al6x/pl0t?subdir=api/nim",
+    "method": "git",
+    "tags": [
+      "plot",
+      "chart",
+      "table",
+      "excel",
+      "spreadsheet",
+      "visualization",
+      "data"
+    ],
+    "description": "Plot and visualize data",
+    "license": "MIT",
```

Comment:
This doesn't look like MIT: https://github.com/al6x/pl0t/blob/main/license

Reply:
Thanks, updated.